backend/regalloc/regalloc_ls_state.ml

```diff
@@ -39,12 +39,15 @@ let dummy_interval =
   }
 
 let make_class_interval_array () =
-  (* CR-soon xclerc for xclerc: this will essentially use twice the required
-     memory *)
-  { elements = Array.make (Reg.total_registers ()) dummy_interval; length = 0 }
+  { elements = Array.make 32 dummy_interval; length = 0 }
```

Comment:
I think it is a welcome change, but
(as in other places) we should
probably leave a CR to consider
switching to `Dynarray` in due
course.